drivers/sensor/lis2dh/Kconfig

```diff
@@ -138,4 +138,8 @@ config LIS2DH_ODR_9_LOW
 
 endchoice
 
+config LIS2DH_ANYM_ON_INT1
+	bool "Configure any movement to use interrupt 1 instead of 2"
+	depends on LIS2DH_TRIGGER
+
```

Comment:
Can you think about moving this in DT?
Like defining anymotion-int in the dts binding, or something like that.